src/Test/Plutip/Config.hs

```diff
@@ -16,8 +18,11 @@ data PlutipConfig = PlutipConfig
     relayNodeLogs :: Maybe FilePath
   , -- | in case of `Nothing` port from `Plutus.ChainIndex.Config.defaultConfig` is used
     chainIndexPort :: Maybe Natural
+  , -- | Any extra pre-determined signers to use.
+    --    Either provided by a path to the signing key file, or by the signing key itself.
+    extraSigners :: [Either FilePath (SigningKey PaymentKey)]
```

Comment:
[nits] I would prefer some custom made type here instead of `Either` (e.g. `SigningKeyOrFile`). Either is usally used for error handling, so it could get confusing.

Reply:
I'd say `Either` isn't really tied to error handling in specific, though that is a common usecase. It probably could've been named `Result` if its intended usecase was specifically error handling.

With that said, I'm fine with declaring a custom type as well. Do you think something like this could be good?
```hs
data SigningKeyInfo = RawKey (SigningKey PaymentKey) | KeyFile FilePath
```